Dockerfile.app

```diff
@@ -7,16 +7,16 @@ RUN curl -f https://get.pnpm.io/v6.16.js | node - add --global pnpm
 
 WORKDIR /web
 
-# copy all dependency declerations inside the container
+# copy all dependency declarations inside the container
 COPY package*.json .
 COPY pnpm*.yaml .
-COPY app/package*.json ./app/
+COPY tasks/package*.json ./tasks/
 COPY lib/package*.json ./lib/
 
-# install depedencies
+# install dependencies
 RUN pnpm install --frozen-lockfile
 
-# build app
+# build tasks
```

Comment:
`# build tasks app` sounds a bit better

Reply:
As a side note: why is this file called `Dockerfile.app`, shouldn't this just be `Dockerfile` and live in the `/app` directory?

Reply:
Yes i agree

Reply:
Due to the pnpm workspaces.